site/app/templates/GlobalHeader.twig

```diff
@@ -252,14 +252,10 @@
             Manually reload this page to see updates.{% if sysadmin_email is not empty %} Notify the
             sysadmin at <a href='mailto:{{ sysadmin_email }}'>{{ sysadmin_email }}</a>.{% endif %}
         </div>
-        {% include 'Vue.twig' with {
-            'component': 'performanceWarning',
-            'args': {
-                "performanceWarning": performance_warning,
-                "submittyQueries": submitty_queries,
-                "courseQueries": course_queries
-            }
-        } %}
+        <div class="system-message warning" id="performance-warning-system-message" style="display: none;">
+            <span id="performance-warning-system-message-text"></span>
+            <i id="close-performance-warning-system-message" class="fa fa-times" style="padding: 4px; cursor: pointer;"></i>
+        </div>
```

Comment:
It looks like there was a bad conflict resolution here.  All of the changes to this file can just be reverted, right?